packages/components/src/public/QueryModel/FiltersButton.tsx

```diff
@@ -5,25 +5,25 @@ import { Tip } from '../../internal/components/base/Tip';
 
 interface Props {
     onFilter: () => void;
+    iconOnly?: boolean;
 }
 
 export const FiltersButton: FC<Props> = memo(props => {
-    const { onFilter } = props;
+    const { onFilter, iconOnly } = props;
 
-    return (
-        <>
-            <span className="hidden-md hidden-sm hidden-xs">
+    if (iconOnly) {
```

Comment:
I kinda like the old implementation how the button renders based on bootstrap size. Is there a reason to switch to use explicit iconOnly?

Reply:
See comment above. It has to do with the scss for the margin-left for `.button-bar__section > *:not(:first-child)`. It was just easier to make sure only one filter button would be rendered then it was to figure out how to update this css selector.